tests/NHSD.BuyingCatalogue.Identity.Api.UnitTests/Controllers/AccountControllerTests.cs

```diff
@@ -62,14 +67,17 @@ public async Task Login_LoginViewModel_FailedSignIn_AddsDisabledValidationError(
             (string key, ModelStateEntry entry) = modelState.First();
             key.Should().Be(nameof(LoginViewModel.DisabledError));
             entry.Errors.Count.Should().Be(1);
-            entry.Errors.First().ErrorMessage.Should().Be(AccountController.UserDisabledErrorMessage);
+
+            var expected = string.Format(CultureInfo.CurrentCulture, AccountController.UserDisabledErrorMessageTemplate, email,
```

Comment:
One suggestion is to limit the number of assertions per test, hopefully to make tests easier to read and understand. Could a change be made to just compare the actual model state with an expected value? This would mean no need to check the count of the error list as well.